.coveragerc

```diff
@@ -1,6 +1,6 @@
 [run]
 branch = True
-omit = **/test_*, **/*_test*.*, **/__init__.py, **/rules/**
+omit = **/test_*, **/*_test*.*, **/__init__.py, **/section/**
```

Comment:
I'm not sure this will do what you intend. I could be wrong, but I don't think it will match folders like _section21_. I think you need **/section*/**.

Reply:
addressed.